crates/oxc_allocator/src/arena.rs

```diff
@@ -8,155 +8,9 @@ use std::{
     ops, ptr,
 };
 
-use bumpalo::collections;
-pub use bumpalo::collections::String;
+use blink_alloc::BlinkAlloc;
+// use bumpalo::collections;
+// pub use bumpalo::collections::String;
 use serde::{ser::SerializeSeq, Serialize, Serializer};
 
 use crate::Allocator;
-
-/// Bumpalo Box
-pub struct Box<'alloc, T: ?Sized>(pub &'alloc mut T);
-
-impl<'alloc, T> Box<'alloc, T> {
-    #[must_use]
-    pub fn unbox(self) -> T {
-        // This pointer read is safe because the reference `self.0` is
-        // guaranteed to be unique--not just now, but we're guaranteed it's not
-        // borrowed from some other reference. This in turn is because we never
-        // construct an alloc::Box with a borrowed reference, only with a fresh
-        // one just allocated from a Bump.
-        unsafe { ptr::read(self.0 as *mut T) }
-    }
-}
```

Comment:
I kind of liked the `unbox` API, its from jsparagus

https://github.com/mozilla-spidermonkey/jsparagus/blob/939577de8fb80c94c4a1595a2be69802327be680/crates/ast/src/arena.rs#L13-L22

Reply:
`Box::into_inner` is from `std`.
And `Box` should not have methods with `self` to not cause name resolution conflict with pointee.